organisaatio-ui/src/components/Modaalit/ToimipisteenNimenmuutos/constants.tsx

```diff
@@ -0,0 +1,2 @@
+export const MUUTOSTYYPPI_EDIT = 'EDIT';
```

Comment:
do we need these when the values are typed?

Reply:
these are used in many places, so i figured it would be good to use the same constants.